scripts/updates/update_publications.py

```diff
@@ -17,6 +17,10 @@
 db.Parallaxes.update().where(db.Parallaxes.c.reference == 'Liu16').values(reference='Liu_16').execute()
 db.Publications.delete().where(db.Publications.c.publication == 'Liu16').execute()
 
+# Removing accents
+db.Publications.update().where(db.Publications.c.publication == 'Góme01').values(publication='Gome01').execute()
+db.Publications.update().where(db.Publications.c.publication == 'Lópe04').values(publication='Lope04').execute()
```

Comment:
I'm not convinced. I kinda want to leave these. The accents aren't causing any problems.

Reply:
The accents are causing problems- they can flip to unicode designations in some machines depending on the order of operations (which is what happened on the other PR)

Reply:
ok, I won't die on this hill.